code/extensions/PayPalRestExtension.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * PayPal Rest can work without taking a credit card, this extension deals with
+ * that and the response that comes back from PayPal
+ *
+ * # payment.yml
+ * SilverStripe\Omnipay\Service\PaymentService:
+ *  extensions:
+ *    - PayPalRestExtension
+ */
+class PayPalRestExtension extends Extension
+{
+    /**
+     * @param array $gatewayData
+     */
+    public function onBeforePurchase(array &$gatewayData)
+    {
+        /**
+         * If you don't want to supply a Credit card then unsetting this will
+         * take you through to PayPal without one
+         */
+        unset($gatewayData['card']);
```

Comment:
I've also tried this locally, and indeed, it seems to be a good idea to remove the card… since you'll be redirected to the offsite payment form anyway and according to the DocBlock, using a card is restricted to US and UK only.

You'd need to make sure this will only happen for the `PayPal_Rest` gateway though… eg. use something like:

    if ($this->owner->getPayment()->Gateway === 'PayPal_Rest') {
        unset($gatewayData['card']);
    }